local-dev/rabbitmqqueues.bicep

```diff
@@ -17,17 +17,14 @@ limitations under the License.
 @description('Information about what resource is calling this Recipe. Generated by Radius. For more information visit https://docs.radapp.dev/operations/custom-recipes/')
 param context object
 
-@description('Name of the queue. Defaults to the name of the Radius RabbitMQ resource.')
-param queue string = context.resource.name
-
 @description('Tag to pull for the rabbitmq container image.')
 param tag string = '3'
 
 @description('Memory request for the rabbitmq deployment.')
-param memoryRequest string = '256Mi'
+param memoryRequest string = '512Mi'
 
 @description('Memory limit for the rabbitmq deployment')
-param memoryLimit string = '1024Mi'
+param memoryLimit string = '2048Mi'
```

Comment:
Why did we need to increase the limits? Increasing the memory request should be fine in my opinion. Do we need to add any limits and/or requests for the CPU?

Reply:
I was testing something while testing the recipe, there is no need to increase the limits. I will revert these changes back

Reply:
Were they reverted?